docs/organization/dynamic-sampling/index.mdx

```diff
@@ -1,18 +1,54 @@
 ---
-title: Retention Priorities
-sidebar_order: 130
-description: "Learn how Sentry determines retention priorities and how to update them."
+title: Dynamic Sampling
+sidebar_order: 30
+description: "Learn how to make your billing quota go farther with Sentry's Dynamic Sampling."
 ---
 
+## Overview
+
+High-volume projects that generate many events can quickly consume a large portion of your billing quota, leaving less room for other projects and decreasing visibility across them.
+
+With high-volume projects, you'll often see many similar events, which can be redundant and less helpful for debugging. But with Dynamic Sampling, you can maximize your quota by adjusting sample rates based on each project’s event volume. The sample rate is a percentage from 0% (no events passed) to 100% (all events passed), that can be tailored to each project’s needs.
```

Comment:
This might be misleading. Without going into the details, there's always going to be a quota for how much is sent to Sentry by the SDKs. Dynamic Sampling has no effect on that, so you cannot just "maximize your quota". Sent spans is always going to be the smaller price component, so it makes sense to move more of the sampling to Sentry. The benefit of that is that Dynamic Sampling is smarter and adapts to changes, so you generally will get higher quality samples than with the random sampling in SDKs. We can further call out some examples from the priorities, like automatically boosting latest releases or boosting small endpoints.